minimal-plugin/src/main/java/io/micronaut/gradle/MicronautExtension.java

```diff
@@ -107,6 +117,32 @@ public MicronautExtension runtime(MicronautRuntime micronautRuntime) {
         return this;
     }
 
+    /**
+     * Configures the application type.
+     *
+     * @param applicationType The micronaut application type
+     * @return This extension
+     */
+    public MicronautExtension applicationType(String applicationType) {
```

Comment:
I would personally avoid those helper methods, which are not Gradle idiomatic.

Reply:
Please, note that this methods exists for `runtime` as well. That it is why I created it. Should I remove both?

Reply:
Deleted in this commit https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/424/commits/9ff5488843f9fc4abeb8dd21ed33cd42f52adccf